spec/spec_helper.rb

```diff
@@ -70,5 +70,6 @@ def self.execute(request)
   config.infer_spec_type_from_file_location!
 
   include FactoryGirl::Syntax::Methods
```

Comment:
We can just delete this. We always call FactoryGirl methods with FactoryGirl.whatever anyways